components/PaginatedSuppliers/Paginated.module.scss

```diff
@@ -0,0 +1,106 @@
+@import 'colors','fonts','layout';
+
+
+.relative {
+  position: relative;
+}
+
+.container {
+  bottom: calc(-82% + 40px);
```

Comment:
there should be no hover yet. I suggest that the parent component be relative 
![chrome_VWRZnwr5dS](https://user-images.githubusercontent.com/71513679/188704341-790f2f28-be7e-4cf6-81a6-d1077d929593.gif)

Reply:
Currently, it's using the list as a reference and then moving "-82%" to the bottom. This could create unexpected behaviors, here's an example when if exist more than 3 items the pagination is moved out of the screen.

![lastrecord](https://user-images.githubusercontent.com/66505715/188711328-c10922b5-100b-4b3b-a523-8b011bed3f2b.gif)

Reply:
A suggestion just in case:

I'd suggest adding `position: relative` to the `main` element

![imagen](https://user-images.githubusercontent.com/66505715/188712529-2e94f2fc-b9a6-4a5b-a862-899d969b2c83.png)

Remove this position relative from the providers list container

![imagen](https://user-images.githubusercontent.com/66505715/188712575-e430a488-5028-4152-bd7a-55a5594d4bfd.png)

Remove some classes in the Paginated_paginated-container element, and just use it to position the elements with flex:

![imagen](https://user-images.githubusercontent.com/66505715/188712991-06c09b38-d4b1-4892-bb8c-4a674ab8aa05.png)

And position the component:
```css
bottom: /** define here how close do you want to have the element from the bottom, I used 15px but please check on the design the correct space */
right: /** define here how close do you want to have the element from the right, I used 30px but please check on the design the correct space */
left: /**define here how close do you want to have the element from the left navbar, I used 15px but please check on the design the correct space */
z-index: /** define here the z-index of you element, I used 100 but it could be 1 or 10 hehe */ 
```
![imagen](https://user-images.githubusercontent.com/66505715/188713148-64a7a483-bc46-4877-bee1-ca73c6bc52b3.png)